lib/ansible/module_utils/common/notice.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Ansible Project
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+from ansible.module_utils.six import string_types
+
+global_warnings = []
+global_deprecations = []
+global_debugs = []
```

Comment:
I'm not crazy about this name. It seemed to fit with the other names though `global_debug_messages` seems less janky.